viewer/package.json

```diff
@@ -64,8 +64,8 @@
   },
   "devDependencies": {
     "@azure/msal-browser": "2.24.0",
-    "@cognite/sdk": "7.5.1",
-    "@cognite/sdk-core": "^4.1.1",
+    "@cognite/sdk": "7.6.1",
+    "@cognite/sdk-core": "4.4.0",
```

Comment:
Do we want to keep ^ for minor versions update?

Reply:
No, don't trust them anymore. So need to validate even minor versions...